packages/bull-vault/src/FlashBull.sol

```diff
@@ -67,16 +67,19 @@ contract FlashBull is UniFlash {
         uint256 ethToDepositInCrab;
     }
 
+    /// @dev struct for data encoding
```

Comment:
I think this should maybe match the comment for `FlashDepositCrabData` as they are all the same between all of these? Or should be more specific and unique for each one.